src/voice/audio.rs

```diff
@@ -36,12 +40,53 @@ pub enum AudioType {
 /// Control object for audio playback.
 ///
 /// Accessed by both commands and the playback code -- as such, access is
-/// always guarded.
+/// always guarded. In particular, you should expect to receive
+/// a [`LockedAudio`] when calling [`Handler::play_returning()`] or
+/// [`Handler::play_only()`].
+///
+/// # Example
+/// ```rust, no-run
+/// use serenity::voice::{Handler, LockedAudio, ffmpeg};
+///
+/// let handler: Handler = /* ... */;
+/// let source = ffmpeg("../audio/my-favourite-song.mp3")?;
+/// let safe_audio: LockedAudio = handler.play_only();
+/// {
+///     let audio_lock = safe_audio_control.clone();
+///     let mut audio = audio_lock.lock();
+///
+///     audio.volume(0.5);
+/// }
+/// ```
+/// [`LockedAudio`]: type.LockedAudio.html
+/// [`Handler::play_only()`]: struct.Handler.html#method.play_only
+/// [`Handler::play_returning()`]: struct.Handler.html#method.play_returning
 pub struct Audio {
+    /// Whether or not this sound is currently playing.
+    /// Can be controlled with [`.play()`] or [`.pause()`]
```

Comment:
Let's just name this in the style of "[\`play\`]", along with all other function references.